scripts_tests/test_aster.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+import shutil
+from pathlib import Path
+import pytest
+from datacube.index.hl import Doc2Dataset
+
+from scripts.index_aster_lpdaac import generate_lpdaac_defn, generate_lpdaac_doc, generate_vrt, selected_bands
+from scripts.index_aster_lpdaac import raster_to_measurements, vrt_file_path
+
+from digitalearthau.testing import factories
+from digitalearthau.testing.plugin import local_config, integration_config_paths, INTEGRATION_DEFAULT_CONFIG_PATH
+module_db = factories.db_fixture("local_config", scope="module")
+module_index = factories.index_fixture("module_db", scope="module")
+module_dea_index = factories.dea_index_fixture("module_index", scope="module")
+
+SCRIPTS_TEST_DATA = Path(__file__).parent / 'data'
+
+PRODUCTS = {'aster_l1t_vnir': {'ImageData2', 'ImageData1', 'ImageData3N'},
+            'aster_l1t_tir': {'ImageData10', 'ImageData11', 'ImageData12', 'ImageData13', 'ImageData14'}}
+
+EXTRA_METADATA_PREFIXES = {
```

Comment:
Wouldn't it make more sense to include the same dictionaries from the main script, instead of redefining them?

Reply:
Yep Done